dashboard/app/models/activity_section.rb

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: activity_sections
+#
+#  id                 :integer          not null, primary key
+#  lesson_activity_id :integer          not null
+#  position           :integer          not null
+#  properties         :string(255)
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#
+
+class ActivitySection < ApplicationRecord
+  include SerializedProperties
+
+  belongs_to :activity, class_name: 'LessonActivity', foreign_key: :lesson_activity_id, inverse_of: :activity_sections
```

Comment:
these options are added so that we can call this association `.activity` instead of `.lesson_activity`. (on this model and on `Lesson`)

Reply:
Is this really worth the added complexity and maybe confusion?